packages/react-interactions/events/src/dom/create-event-handle/useEvent.js

```diff
@@ -29,14 +29,12 @@ export default function useEvent(
     priority?: 0 | 1 | 2,
   |},
 ): UseEventHandle {
-  const handleRef = useRef(null);
-  let setListener;
-  let clears;
-  let useEventHandle;
+  const handleRef = useRef<UseEventHandle | null>(null);
```

Comment:
Any reason we use generics for the event and target types internally, but not in this repo?

Reply:
We use Scopes internally, which means that the target needs be generic to cover them too. On the web, we can stick with EventTarget, so I could simplify it on githib. Good question though!